tests/ltp/umount02.patch

```diff
@@ -0,0 +1,81 @@
+diff --git a/testcases/kernel/syscalls/umount/umount02.c b/testcases/kernel/syscalls/umount/umount02.c
+index cd67aa6d2..8822eab89 100644
+--- a/testcases/kernel/syscalls/umount/umount02.c
++++ b/testcases/kernel/syscalls/umount/umount02.c
+@@ -20,6 +20,11 @@
+ #include "tst_test.h"
+ 
+ #define MNTPOINT	"mntpoint"
++const char* src  = "/dev/vda";
```

Comment:
Add a comment somewhere that explains why this patch is necessary so that it can be understood in the future.

Reply:
will do